hack/run-wmcb-ci-e2e-test.sh

```diff
@@ -57,6 +57,17 @@ fi
 sed -i "s~ARTIFACT_DIR_VALUE~${ARTIFACT_DIR}~g" internal/test/wmcb/deploy/job.yaml
 sed -i "s~REPLACE_IMAGE~${WMCB_IMAGE}~g" internal/test/wmcb/deploy/job.yaml
 
+DOCKER_RUNTIME=${DOCKER_RUNTIME:-}
+
+# If DOCKER_RUNTIME is set, set the value for flag -dockerRuntime in job.yaml.
+# If it is unset, remove the line with -dockerRuntime flag to avoid passing
+# flag with nil value. Default value for the flag will be true in this case.
+if [ ! -z "$DOCKER_RUNTIME" ]; then
+  sed -i "s~DOCKER_RUNTIME~${DOCKER_RUNTIME}~g" internal/test/wmcb/deploy/job.yaml
+else
+  sed -i '/-dockerRuntime=/d' internal/test/wmcb/deploy/job.yaml
+fi
```

Comment:
```suggestion
DOCKER_RUNTIME=${DOCKER_RUNTIME:-true}

sed -i "s~DOCKER_RUNTIME~${DOCKER_RUNTIME}~g" internal/test/wmcb/deploy/job.yaml
```
Could this simplified by providing a default value?

Reply:
yes I proposed that in the first revision of the PR but changed after I received a comment from Jose pointing out the default value would be true even if the flag is not passed. I am fine with either approach whichever is easier to read.

Reply:
updated